src/inference/src/threading/ie_cpu_streams_executor.cpp

```diff
@@ -107,13 +116,23 @@ struct CPUStreamsExecutor::Impl {
                                 return p.second > streamId_wrapped;
                             })
                             ->first;
-                    const auto max_concurrency = selected_core_type == 0 ? _impl->_config._threads_per_stream_small
-                                                                         : _impl->_config._threads_per_stream_big;
+                    const auto max_concurrency =
+                        hybrid_core ? (selected_core_type == 0 ? _impl->_config._threads_per_stream_small
+                                                               : _impl->_config._threads_per_stream_big)
+                                    : _impl->_config._threads_per_stream_big;
                     const auto stream_id =
-                        selected_core_type == 0 ? _streamId - _impl->_config._big_core_streams : _streamId;
-                    const auto thread_binding_step = selected_core_type == 0 ? _impl->_config._threadBindingStep : 2;
+                        hybrid_core ? (selected_core_type == 0
+                                           ? streamId_wrapped - _impl->_config._big_core_streams
+                                           : (cur_logic_core ? streamId_wrapped - phy_core_streams : streamId_wrapped))
+                                    : streamId_wrapped;
+                    const auto thread_binding_step =
+                        hybrid_core ? (selected_core_type == 0 ? _impl->_config._threadBindingStep : 2)
+                                    : _impl->_config._threadBindingStep;
                     // Prevent conflicts with system scheduling, so default cpu id on big core starts from 1
-                    const auto cpu_idx_offset = selected_core_type == 0 ? _impl->_config._small_core_offset : 1;
+                    const auto cpu_idx_offset =
+                        hybrid_core
+                            ? (selected_core_type == 0 ? _impl->_config._small_core_offset : (cur_logic_core ? 0 : 1))
+                            : 0;
```

Comment:
Need addtional stream start point. 

For example, 8Ecore assign to 2 stream. Stream 1 should use Ecore 1, 2, 3. Stream 2 should use Ecore 5, 6 7